posts/posts-router.js

```diff
@@ -0,0 +1,51 @@
+const express = require('express')
+const db = require('../data/db');
+const port = 4001;
+const router = express.Router();
+
+router.post("/", (req, res) => {
+    const postInfo = req.body;
+  
+    if (!postInfo.title || !postInfo.contents) {
```

Comment:
There we go! Nice conditional. Only problem here is that you will send back an error to the user, but the post will still go through if they just have a `.title` in the req.body. 
We need to wrap our db call inside an else block.